tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -1430,10 +1412,7 @@ public function testSitePreviewBranding(): void
         $proxyClient = new Client();
         $proxyClient->setEndpoint('http://' . $previewDomain);
 
-        $response = $proxyClient->call(Client::METHOD_GET, '/', array_merge([
-            'content-type' => 'application/json',
```

Comment:
Why did we remove the headers from tests?

Reply:
They were not nessessary, visits from browser on site domains dont usually have those, so best to test with same scenarios